examples/08-DeepAR-examples.ipynb

```diff
@@ -11,7 +11,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        series=day_scaled,
</pre><p>this should be </p><pre class="ql-syntax" spellcheck="false">series=series_en_transformed,
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>